### PR TITLE
docs: update quickfact template [skip chromatic]

### PR DESCRIPTION
**Pull request by auroraVasconcelos**
## Description:
Closes https://github.com/solid-design-system/solid/issues/1449.

## Definition of Reviewable:

- [x] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated

- [x] Stories (features, a11y) are created/updated
- [ ] relevant tickets are linked

### CODE REVIEW

File: packages/components/src/templates/quickfact.stories.ts [hunk 1]
```diff
@@ -16,6 +16,19 @@ export default {
...
+      <sd-icon name="content/people" color="primary" aria-hidden="true" library="default" slot="icon"></sd-icon>
+      <div slot="summary" class="flex flex-col sm:gap-4">
+        <div class="text-base font-normal leading-normal sm:text-xl">
+          Over 4,000 employees in 9 international locations
+        </div>
+      </div>
+    </sd-quickfact>`
+};
+
 export const QuickfactWithDescription = {
```
Comment (karlbaumhauer) [inline packages/components/src/templates/quickfact.stories.ts:32]: looking at the figma file mentioned in the issue this PR is about, one template is still missing:
image

is this intended?

Comment (karlbaumhauer) [reply packages/components/src/templates/quickfact.stories.ts:32]: we don't have this variant at all in storybook... :/

**Review by karlbaumhauer — CHANGES_REQUESTED** [review]

File: packages/components/src/templates/quickfact.stories.ts (hunk 1 above)
Comment (auroraVasconcelos) [reply packages/components/src/templates/quickfact.stories.ts:32]: Hey @karlbaumhauer, just use the dev tools to see the mobile versions :) 

image

**Review by karlbaumhauer — APPROVED** [review]

File: packages/components/src/templates/quickfact.stories.ts [hunk 2]
```diff
@@ -16,6 +16,19 @@ export default {
   title: 'Templates/Quickfact'
 };
 
+export const QuickfactWithSummary = {
+  name: 'Quickfact with Summary',
+  render: () =>
+    html` <sd-quickfact class="max-w-[410px]">
+      <sd-icon name="content/people" color="primary" aria-hidden="true" library="default" slot="icon"></sd-icon>
+      <div slot="summary" class="flex flex-col sm:gap-4">
```
Comment (smfonseca) [inline packages/components/src/templates/quickfact.stories.ts:24]: I think these classes are not needed and can be removed.

File: packages/components/src/templates/quickfact.stories.ts [hunk 3]
```diff
@@ -16,6 +16,19 @@ export default {
   title: 'Templates/Quickfact'
 };
 
+export const QuickfactWithSummary = {
+  name: 'Quickfact with Summary',
+  render: () =>
+    html` <sd-quickfact class="max-w-[410px]">
```
Comment (Vahid1919) [inline packages/components/src/templates/quickfact.stories.ts:22]:
```suggestion
    html` <sd-quickfact class="max-w-[410px] mx-auto">
```

This centers the first quickfact making it consistent with the remaining quickfacts and the figma docs.

**Review by karlbaumhauer — APPROVED** [review]

**Review by yoezlem — APPROVED** [review]

**Review by smfonseca — APPROVED** [review]

**Review by MartaPintoTeixeira — APPROVED** [review]

**Comment by MartaPintoTeixeira** [comment]
As spoken please fix the font sizes.

**Comment by auroraVasconcelos** [comment]
> As spoken please fix the font sizes.

Updated

**Pull request merged**